ggml/CMakeLists.txt

```diff
@@ -146,7 +146,7 @@ option(GGML_METAL_EMBED_LIBRARY             "ggml: embed Metal library"
 set   (GGML_METAL_MACOSX_VERSION_MIN "" CACHE STRING
                                             "ggml: metal minimum macOS version")
 set   (GGML_METAL_STD "" CACHE STRING       "ggml: metal standard version (-std flag)")
-option(GGML_OPENMP                          "ggml: use OpenMP"                                ON)
+option(GGML_OPENMP                          "ggml: use OpenMP"                                OFF)
```

Comment:
I think OpenMP should remain the default for now.

Reply:
The thinking was that the threadpool is as/or more efficient now and with more finegrain controls (ie batch/non-batch cpumask, etc), and it remove libomp dependency. We can flip it back though.